elementary/clients/dbt/dbt_runner.py

```diff
@@ -16,8 +16,8 @@ class DbtRunner:
     def __init__(
         self,
         project_dir: str,
-        profiles_dir: str,
-        target: Optional[str] = None,
+        profiles_dir: str = None,
+        target: str = None,
```

Comment:
Well those are optional vars, so why remove the Optional?
It is really helpful to have a good typing!